Code-4-Community/hands-across-the-sea-frontend
# Moved back button outside FormContentContainer

## Why

https://app.clickup.com/t/1tmncmr



Move the back button above the form title to make it consistent with the appearance on other pages. 

## This PR



Moved the BackButton component out of the container. 

Some checks were failing following the merge of the react query refactor. I've updated the data visualization container to use react query to fix these issues. 

## Screenshots

 

<img width="1097" alt="Screen Shot 2021-12-08 at 10 27 10 PM" src="https://user-images.githubusercontent.com/56360563/145329166-bef5ba28-4b41-4533-84b9-d64fc27f2520.png">
<img width="1553" alt="Screen Shot 2021-12-14 at 6 22 41 PM" src="https://user-images.githubusercontent.com/56360563/146095111-51ba4926-557c-48a0-9d40-3e5a5960eb97.png">


## Verification Steps

 

Select the Form History button on the home page. Notice how the back button is above the title. 
Repeat above steps for the Fill Out A Form button.
